pkgs/development/tools/heroku/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, postgresql, ruby }:
+
+stdenv.mkDerivation rec {
+  version = "3.30.3";
+  name = "heroku-${version}";
+
+  src = fetchurl {
+    url = "https://s3.amazonaws.com/assets.heroku.com/heroku-client/heroku-client-${version}.tgz";
+    sha256 = "0m9l04syli4ripkh37lwk0hq4silnp830ddsk3ph77iymzh2iz1f";
+  };
+
+  installPhase = ''
+    mkdir -p $out
+    cp -R * $out/
+  '';
+
+  buildInputs = [ ruby postgresql ];
```

Comment:
@mirdhyn The heroku toolbelt features commands to access your application's database etc.